### ISSUE TITLE
Tertiary text colour has insufficient color contrast

### ISSUE THREAD

**Issue by JezDriver** [xdesro/personalsit.es#1201]
The tertiary text colour (`#8a8a8a`) has poor contrast against the background (`#f7f7f7`).

It fails the WCAG contrast test except for large text, but it's used for the 'TAGS' heading and 'GO TO FEED' link, both of which are `16px` — not large.
https://webaim.org/resources/contrastchecker/?fcolor=8A8A8A&bcolor=F7F7F7

The secondary text colour (`#6b6b6b`) just passes WCAG AA, so an easy win would be to use `--color__text--secondary` instead.

---

I've added this as an issue rather than a PR, since:
- the graphic design champions may want to choose a different way to de-emphasise those bits of text, and
- maybe this is already known (it probably does help readability that these bits of text are bold and all-caps)

xoxo

**Comment by DeclanChidlow** [comment]
Rectified. Thanks Jez!
